probes/securityPolicyContainsLinks/def.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: securityPolicyContainsLinks
+short: Check that the security policy contains web or email links.
+motivation: >
+  URLs point users to additional information as well as online disclosure forms. Emails provide a point of contact for vulnerability disclosure.
+implementation: >
+  The implementation looks for strings "http(s)://" to find URLs; and for strings "...@..." for email addresses.
+outcome:
+  - If links are found, one finding with OutcomePositive (1) is returned for each file.
+  - If no links are found, one finding with OutcomeNegative (0) is returned for each file.
+  - If no file is found, one finding with OutcomeNegative (0) is returned.
```

Comment:
would this be NotApplicable if we added it later?

Reply:
I think so yes.

Reply:
Do we have any tracking issue for discussing probe outcomes?

Reply:
The closest is https://github.com/ossf/scorecard/issues/2928